### ISSUE TITLE
CREO2URDF – Add the possibility to export meshes in formats different from stl

### ISSUE THREAD

**Issue by Nicogene** [icub-tech-iit/creo2urdf#61]
### Task description

We would like to add the possibility of exporting the meshes in formats different from stl
The format chosen probably has to be set on the yaml

### DoD

We can export meshes in different formats

**Comment by Nicogene** [comment]
I started working on this activity but it seems more complicated than expected.

It seems that CREO at some point migrated how the meshes can be exported, in fact, there are plenty of deprecated functions, but unfortunately, right now we cannot use the new ones.

Until now we used this function:

https://github.com/icub-tech-iit/creo2urdf/blob/ed6b2d18067446fe6bf8179740bd703e84293df7/src/creo2urdf/src/Creo2Urdf.cpp#L588

It needs a `pfcExportInstructions` pointer and there is a type of export instruction for every type of format but for some reason some of them are deprecated. For example the `pfcSTEP3DExportInstructions` is deprecated, the documentation suggest to use

```c++
component_handle->ExportIntf3D(mesh_file_name.c_str(), pfcExportType::pfcEXPORT_STEP);
```
instead, and this is fine, since there is a `pfcExportType` for every format, but unfortunately  `ExportIntf3D` does:
```
Exports a Creo Parametric model using default export profile
```

in our config.pro we have only an export profile for the `step` files:

![immagine](https://github.com/icub-tech-iit/creo2urdf/assets/19152494/a93e1930-1ff4-4606-a435-291792566224)

Invoking that function for neutral and stl fails.
This means that right now we have to have a mixed situation, where we use the `model->Export` functions for stl and neutral, and `model->ExportIntf3D` for the step.

I don't see great problems here, I want just to make clear the choice of using different methods for basically doing the same thing.

cc @mfussi66 @pattacini @fiorisi
